packages/opentelemetry-instrumentation-openai/tests/traces/test_functions.py

```diff
@@ -138,10 +183,6 @@ async def test_open_ai_function_calls_tools_streaming(
         open_ai_span.attributes[f"{SpanAttributes.LLM_COMPLETIONS}.0.tool_calls.0.id"],
         str,
     )
-    assert (
```

Comment:
I dropped these checks because these tests are checking the LLM_COMPLETIONS output and the functionality for checking LLM_TOOLS shouldn't differ